riak_test/yz_startup_shutdown.erl

```diff
@@ -134,3 +141,29 @@ verify_drain_called(Cluster) ->
                              ok =:= Result
                      end,
                      Results).
+
+%% @private
+%% @doc For each node in `Cluster', verify that the ibrowse configuration has
+%% been applied.
+verify_ibrowse_config([Node1|_] = Cluster) ->
+    {ResL, []} = rpc:multicall(Cluster, yz_solr, get_ibrowse_config, []),
+    lists:foreach(
+      fun(Config) ->
+              MaxSessions = proplists:get_value(max_sessions, Config),
+              MaxPipelineSize = proplists:get_value(max_pipeline_size, Config),
```

Comment:
@zeeshanlakhani I can add a couple of macros to `yokozuna.hrl` and use them. Any preference for naming? I'm thinking one of the following naming schemes:

```erlang
-define(IBROWSE_MAX_PIPELINE_SIZE, max_pipeline_size).
-define(YZ_SOLR_IBROWSE_MAX_PIPELINE_SIZE, max_pipeline_size).
-define(YZ_SOLR_MAX_PIPELINE_SIZE, max_pipeline_size).
-define(MAX_PIPELINE_SIZE, max_pipeline_size).
```